client/my-sites/term-tree-selector/add-term.jsx

```diff
@@ -193,6 +207,15 @@ class TermSelectorAddTerm extends Component {
 							onKeyUp={ this.boundValidateInput } />
 						{ isError && <FormInputValidation isError text={ this.state.error } /> }
 					</FormFieldset>
+					{ showDescriptionInput && <FormFieldset>
+							<FormLegend>
+								{ translate( 'Description', { context: 'Terms: Term description label' } ) }
+							</FormLegend>
+							<FormTextarea
```

Comment:
I'm not able to edit the `slug` in wp-admin. I can see it on the Categories/Tags pages but I'm not able to change it directly. but yes we can change the boolean to `showExtendedInputs`.